addons/cat-blocks/userscript.js

```diff
@@ -1,4 +1,7 @@
 export default async function ({ addon, global, console }) {
+  const now = new Date().getTime() / 1000;
+  // April fools
+  if (now < 1617364800 && now > 1617192000) return;
```

Comment:
We don't know when the ST will decide to enable/disable april fools, might not match our schedule